ultralytics/engine/trainer.py

```diff
@@ -299,19 +327,23 @@ def _do_train(self, world_size=1):
         self.epoch_time = None
         self.epoch_time_start = time.time()
         self.train_time_start = time.time()
-        self.run_callbacks('on_train_start')
-        LOGGER.info(f'Image sizes {self.args.imgsz} train, {self.args.imgsz} val\n'
-                    f'Using {self.train_loader.num_workers * (world_size or 1)} dataloader workers\n'
-                    f"Logging results to {colorstr('bold', self.save_dir)}\n"
-                    f'Starting training for '
-                    f'{self.args.time} hours...' if self.args.time else f'{self.epochs} epochs...')
+        self.run_callbacks("on_train_start")
+        LOGGER.info(
+            f'Image sizes {self.args.imgsz} train, {self.args.imgsz} val\n'
+            f'Using {self.train_loader.num_workers * (world_size or 1)} dataloader workers\n'
+            f"Logging results to {colorstr('bold', self.save_dir)}\n"
+            f'Starting training for '
+            f'{self.args.time} hours...'
```

Comment:
These can get changed to double quotes. Wonder if the new line/indentation layout caused the quotes replacement get bypassed.